### ISSUE TITLE
Integrate/interoperate with built-in NPM extension

### ISSUE THREAD

**Issue by aaronadamsCA** [microsoft/vscode-js-debug#782]
The overlapping, non-integrated functionality between this built-in extension and the [built-in NPM extension](https://github.com/microsoft/vscode-npm-scripts) makes for a pretty bad user experience.

The biggest problem is the lack of a single (working) interface to run or debug a script. You don't provide run commands; the other extension provides run and debug commands, but their debug command doesn't work the same as yours ([and seems to be broken in monorepo packages](https://github.com/microsoft/vscode-npm-scripts/issues/138)). So for me, only one of these debug buttons actually opens the debugger:

![image](https://user-images.githubusercontent.com/1505561/94936493-9ab10a80-04ce-11eb-906f-8fb28d247556.png)

It's not a great out-of-the-box experience, and I think it would be great if you could work with the other extension (or if needed, take it over) to improve on this.

**Comment by connor4312** [comment]
Both "debug" actions should open the debugger and, as of the last release (1.49) should run the same command. Can you share the failure you're experiencing?

**Comment by aaronadamsCA** [comment]
Sure thing, I described it in microsoft/vscode-npm-scripts#138 - let me know if you want/need more info, here or there. It's specifically happening only when I debug from `/workspaces/*/package.json`, debugging from the root `package.json` seems to work fine.

**Comment by connor4312** [comment]
Thanks, I've fixed the linked issue. From what I can tell that was the only issue you were running into? Both the script lens provided by this package and the built-in npm extension will invoke this debugger.

However it would make sense to move the script lens to be implemented in the npm extension at some point. Reduces some duplicated work and keeps us from needing to activate js-debug as often.

**Comment by aaronadamsCA** [comment]
I am guessing that was probably the only issue, yes! I (wrongly) assumed they weren't integrated since the buttons didn't do the same things. If the debug buttons will work in the NPM Scripts pane, I will be very happy. Thanks!

**Comment by aaronadamsCA** [comment]
Thanks @connor4312. I grabbed Insider and I can confirm the Debug button works, but note its behaviour is not what I'd expect:

1. When I click **Debug** in the `package.json` code lens, the Terminal opens and displays process output 👍
2. When I click **Debug** in the NPM Scripts pane, the Debug Console opens and does not display process output 👎

I think the default behaviour for both entry points should be the same, and I think behaviour 1 is preferable, since it's more similar to Run behaviour + you don't lose the process output.

I found a bunch of similar-ish tickets under vscode, so I'm not going to file a new one; this is more just an "FYI this wasn't quite what I expected." Still glad to have the fix though, thanks.
